site/docs/spec.md

```diff
@@ -159,6 +161,13 @@ A **`list`** is a collection of values with some element type. The element field
 
 A **`map`** is a collection of key-value pairs with a key type and a value type. Both the key field and value field each have an integer id that is unique in the table schema. Map keys are required and map values can be either optional or required. Both map keys and map values may be any type, including nested types.
 
+Iceberg supports default-value semantics for fields of nested types (i.e., struct, list and map). Specifically, a field 
```

Comment:
What does it mean for a list element to have a default value? Similarly, what does it mean for a map value to have a default?

I don't think that list elements or map values are places where we should allow default values. I'm not aware of a case where there is a file that contains a map, but the value column is missing. And I think that's when we would fill in default values.

Reply:
Yes, I just want to make sure that's the goal. No default elements, key/value pairs, or custom lookup results (like `getOrDefault`). Just default values for whole lists, like an empty map or empty list. Maybe a specific map or list that is non-empty?